participants/Jazmine.xml

```diff
@@ -1,5 +1,13 @@
+<<<<<<< HEAD
+<participant>
+  <name>Jazmine Smith</name>
+  <favoriteCereal>Frosted Flakes</favoriteCereal>
+  <funFact>I love steak</funFact>
+</participant>
+=======
```

Comment:
hey jasmine, can you delete:
 the lower section of this file,
 the <<<<<<<<< HEAD,
the ========= and the 
>>>>>>>>> f152d92687454df13d2345f54aedce2ed017041e
it should look like: 

````
<participant>
  <name>Jazmine Smith</name>
  <favoriteCereal>Frosted Flakes</favoriteCereal>
  <funFact>I love steak</funFact>
</participant>
````